src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int top;
+    private int bottom;
+    private int altitude;
+
+    public IsoscelesTrapezoid(String color, int top, int bottom, int altitude) {
+        super(color);
+        this.top = top < bottom ? top : bottom;
+        this.bottom = bottom > top ? bottom : top;
+        this.altitude = altitude;
+    }
+
+    @Override
+    public double getArea() {
+        return (top + bottom) * altitude / 2.0;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(
+                "Figure: "
+                + "Isosceles trapezoid, area: "
+                        + (double) ((long) (getArea() * 10)) / 10 + " sq. units,"
```

Comment:
? there should be no special logic. If you want to round your number, do it in getArea, and for all figures, maybe create a utility method for it in a utility class

Reply:
Sure. It's a good idea. I've just done it